### DISCUSSION TITLE
Implementation of Get/Update Configuration Warnings for C#

### DISCUSSION THREAD

**Discussion by JamieRhys** [godotengine/godot-proposals#8494] [Scripting]
I've started switching over from GDScript to C# and a big thing I've already noticed is that there does not seem to be a method for `get_configuration_warnings()` or `update_configuration_warnings()` version for C#. 

Within my GDScript I use this extensively to prevent any mistakes in my Inspector. 

If someone is able to point me in the general direction, I'd happily have a look at this.

**Comment by AThousandShips** [comment]
What syntax are you using? The names are `_GetConfigurationWarnings` for the overridden method, and `UpdateConfigurationWarnings` respectively, you need it to be `[Tool]`

**Reply by AThousandShips** [reply]
Can you please confirm?

**Reply by JamieRhys** [reply]
You're correct, I just needed the [Tool] :S

**Comment by BlustrawberRi** [comment]
How do you set that up, when you have to make the exported variable a property, so you can call "UpdateConfigurationWarnings" in the Setter when you made a change to it in the inspector (and lose the warning)? Because for me, on every built the value is being forgotten, because I have to write a private field to store the value of the property, to avoid an endless loop.
I do not find a solution to this
